devicon.json

```
@@ -2525,6 +2525,24 @@
           }
         ]
     },
+    {
+        "name": "sqlserver",
+        "tags": [
+            "database"
+        ],
+        "versions": {
+            "svg": [
+                "plain",
+                "plain-wordmark"
+            ],
+            "font": [
+                "plain",
+                "plain-wordmark"
+            ]
+        },
+        "color": "#000000",
```

Maybe they gray color from the logo is better?

```suggestion
        "color": "#909DAA",
```